cosmoz-data-nav.js

```diff
@@ -5,7 +5,23 @@
 	'use strict';
 
 	const _async = window.requestIdleCallback || window.requestAnimationFrame || Polymer.Base.async,
-		_cancelAsync = window.cancelIdleCallback || window.cancelAnimationFrame || Polymer.Base.cancelAsync;
+		_asyncPeriod = function (cb, minimum = 5) {
```

Comment:
We might want to make _asyncPeriod and _asyncAnimation part of cosmoz-behaviors? Feels like something we'd want to reuse?

Reply:
@nomego Not sure about this because in 2.x requestIddleCallback is supported directly in `Polymer.Async` https://www.polymer-project.org/2.0/docs/api/namespaces/Polymer.Async.